quic/handshake/FizzBridge.h

```diff
@@ -9,7 +9,57 @@
 #pragma once
 
 #include <fizz/crypto/aead/Aead.h>
+#include <quic/handshake/Aead.h>
+
+#include <memory>
+#include <utility>
 
 namespace quic {
-using Aead = fizz::Aead;
-}
+
+class FizzAead final : public Aead {
+private:
+  std::unique_ptr<fizz::Aead> fizzAead;
+  FizzAead(std::unique_ptr<fizz::Aead> _fizzAead)
```

Comment:
nit: rename this to fizzAeadIn if you want to avoid collided name. We don't use _someName anywhere else in mvfst.